Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM golang:1.7-alpine
```

Comment:
Does the base image need to have `go` installed? I think the image will also need `iptables` installed since droplan just shells out to `iptables`